docs/translations.rst

```diff
@@ -0,0 +1,310 @@
+
+.. code:: cfg
+
+    # Extraction from Jinja2 HTML templates
+    [jinja2: **/**.html]
+    encoding = utf-8
+    ignore_tags = script,style
+    include_attrs = alt title summary placeholder
+
+
+.. seealso::
+
+    More details check out the following links:
+
+    - `How setup this file <http://babel.pocoo.org/en/latest/setup.html>`_
+    - `A previous file example description <http://babel.pocoo.org/en/latest/messages.html#extraction-method-mapping-and-configuration>`_
+
+Administrative Tasks
+====================
+
+The ``babel`` package have a *Distutils/Setuptools Integration* which supports the options 
+defined in the :file:`setup.cfg` file that can be executed via command line.
+
+These options are the commonly using as **"Translations Administrative Tasks"** and the most 
+used tasks are described below:
+
+Extract messages
+----------------
+
+It can extract localizable messages from a variety of difference source files, 
+and generate a PO (portable object) template file from the collected messages.
+
+Running the following command will produce a PO template file:
+
+.. code:: bash
+
+    $ python ./setup.py extract_messages -o ./sphinx_rtd_theme/locale/sphinx.pot
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#extract-messages
+
+Init catalog
+------------
+
+It creates a new translation catalog based on a PO template file (POT). Running the following 
+command will produce a PO file:
+
+.. code:: bash
+
+    $ python ./setup.py init_catalog -l es -i ./sphinx_rtd_theme/locale/sphinx.pot -o ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po
+    running init_catalog
+    creating catalog './sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po' based on './sphinx_rtd_theme/locale/sphinx.pot'
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#init-catalog
+
+Update catalog
+--------------
+
+It updates an existing translations catalog based on a PO template file (POT). Running the following 
+command will update a PO file:
+
+.. code:: bash
+
+    $ python ./setup.py update_catalog -l es -i ./sphinx_rtd_theme/locale/sphinx.pot -o ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po
+    running update_catalog
+    updating catalog ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po based on ./sphinx_rtd_theme/locale/sphinx.pot
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#update-catalog
+
+Compile catalog
+---------------
+
+It compile catalog an existing translations based on PO files into MO files. Running the following 
+command will compile catalog of PO files:
+
+.. code:: bash
+
+    $ python ./setup.py compile_catalog -d ./sphinx_rtd_theme/locale/ -l es
+    running compile_catalog
+    compiling catalog ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po to ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.mo
+
+Statistics about translations
```

Comment:
Not sure if this is something necessary for our docs. It would be nice if we can write this into our test suite though.

Reply:
@Blendify removed this section :+1: thanks for your feedback :nerd_face: